src/common/components/vanilla-modules/breadcrumbs/index.js

```diff
@@ -0,0 +1,41 @@
+import React, { PropTypes } from 'react';
+import { Link } from 'react-router';
+
+import style from '../../../style/vanilla/css/breadcrumbs.css';
+
+const breadcrumbsStyle = (element = '', modifier = '') => {
```

Comment:
A helper function to generate vanilla BEM class names and bind them to CSS modules `styles` object.